server/src/services/resume.service.js

```diff
@@ -0,0 +1,35 @@
+const httpStatus = require('http-status');
+const { RawExperience, FormattedExperience } = require('../models');
+const OpenAIAdapter = require('../adapters');
+const ApiError = require('../utils/ApiError');
+
+/**
+ * Save a new experience
+ * @param {Object} experienceBody
+ * @returns {Promise<Object>}
+ */
+const saveExperience = async (experienceBody) => {
+  // Check if we already saved experience for the user 
+  if (await RawExperience.RawExperience.userAlreadyHas(experienceBody.userId)) {
+    throw new ApiError(httpStatus.BAD_REQUEST, 'User already has loaded experience');
+  }
+
```

Comment:
Maybe we should also add userId validation? Apparently it can be anything

Reply:
Gonna fix this in new branch `user-auth`